.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,46 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the README at:
+// https://github.com/microsoft/vscode-dev-containers/tree/v0.222.0/containers/python-3-miniconda
+{
+    "name": "Python Environment",
+    "build": {
+        "dockerfile": "Dockerfile",
+        "context": ".."
+    },
+    "customizations": {
+        "vscode": {
+            "extensions": [
+                "amodio.tsl-problem-matcher",
+                "bungcip.better-toml",
+                "cschleiden.vscode-github-actions",
+                "dbaeumer.vscode-eslint",
+                "editorconfig.editorconfig",
+                "GitHub.copilot",
+                "github.vscode-pull-request-github",
+                "ms-azuretools.vscode-docker",
+                "ms-python.python",
+                "ms-python.vscode-pylance",
+                "ms-python.pylint",
+                "ms-python.isort",
+                "ms-python.flake8",
+                "ms-python.black-formatter",
+                "ms-vsliveshare.vsliveshare",
+                "ms-vscode-remote.remote-containers"
+            ],
+            "settings": {
+                "python.defaultInterpreterPath": "/usr/local/bin/python",
+                "black-formatter.path": [
+                    "/usr/local/py-utils/bin/black"
+                ],
+                "pylint.path": [
+                    "/usr/local/py-utils/bin/pylint"
+                ],
+                "flake8.path": [
+                    "/usr/local/py-utils/bin/flake8"
+                ],
+                "isort.path": [
+                    "/usr/local/py-utils/bin/isort"
+                ]
```

Comment:
Are these needed? The bundled version should get used for these.

Reply:
let me try playing with adding and removing them.

I think 'pylint' particularly needs to be setup this way so that it reads import statements correctly...

Reply:
We configure it internally in a way that pylint should find all imports. If pylance can find it, pylint should be able to find it too. If not then that could be a bug.

Reply:
I found a spot to investigate. It looks like pylance doesn't find it either. Let me see if its just missing from requirements.txt

<img width="580" alt="image" src="https://user-images.githubusercontent.com/9027725/225756247-732abe72-efc0-4e1a-b3cb-f0ebab79b1cb.png">

Reply:
That is intentional. `debugpy` should not be installed in any environment unless there is a specifc need. We dynamically inject debugpy when debugging. So this is expected.